### DISCUSSION TITLE
Port to firefox

### DISCUSSION THREAD

**Discussion by skywinder** [cheshire137/soma-chrome#48] [General]
Hi. I would like to build and port it to firefox, if you don't mind, I would like to release it with all the reference to your original repo.

**Comment by cheshire137** [comment]
That suits me fine, thank you! 🙇‍♀ I've taken to using Safari myself recently over Chrome, so I definitely support expanding to more browsers.

If you learn anything while developing the Firefox version that you think could improve this Chrome extension, feel free to let me know via an issue. You're welcome to open a PR to port over any learnings, too, if you'd rather!
